.github/TriageProcess.md

```diff
@@ -0,0 +1,97 @@
+- Set proper expectations with customers regarding how issues are going to be handled
+
+## Triage Process Details
+
+The feature teams should be able look through every single issue filed against this repository and be able to make a quick call regarding the nature of the issue.
+We will first categorize the issues and further handle these depending on the category the issue is in. The subsections below represent these categories and the rules we apply for them during our triage meeting.
+
+### Information Gathering
+
+In this phase we instruct the user on how to collect the appropriate diagnostics and see if they are able to address the issue with that additional information.  When we need user input we will mark the issue with `s/needs-info` label. Issues in this phase may be closed automatically if we do not receive timely responses, they often do not provide enough information for us to investigate further.
+We'll try to respond quickly to such issues (within days). If a user has collected all of the relevant diagnostics and the issue is still not apparent, then we will consider it for further [investigation](#investigations) by the team.
+
+### Feature requests
+
+As soon as we identify an issue represents an ask for a new feature, we will label the issue with the `t/enhancement ☀️` label.
+Most of the time, we will automatically move these issues into the `.NET V Planning` (where V is the .NET version we're planning this for) milestone for further review during one of our [sprint planning meetings](#milestone-planning).
+If we think the feature request is not aligned with our goals, we may choose to close it immediately.
+In some situations, however, we may choose to collect more feedback before acting on the issue. In these situations we will move the issue in the `Backlog` so that we can review it during the [release planning](#release-planning).
+
+### Bug reports
+
+If it's immediately clear, that the issue is related to a bug in the framework, we will apply the `bug` label to the issue.
+
+At this point, we will try to make a call regarding it's impact and severity. If the issue is critical, we may choose to include it in our current milestone for immediate handling or potentially patching.
+If the bug is relatively high impact, we will move the issue into the `.NET V Planning` (where V is the .NET version we're planning this for) milestone to review during our [sprint planning](#milestone-planning) meeting.
+If the impact is unclear or the is a very corner case scenario, we may move it to a next `.NET V Planning` or `Backlog` milestone to further evaluate the impact by reviewing customer upvotes / comments at a later time.
+
+### Investigations
+
+In many situations it's not immediately clear whether a specific issue reported is a bug or not. To be certain, the team will need to spend time to investigate it before making a call regarding the faith of the issue. In these situations we will apply the `investigate` label to the issue.
```

Comment:
Let's slap the investigate label on so we know we need to investigate more first, this can then happen in a sprint rather than at triage.